sycophancy_analysis/dashboard/app.py

```
@@ -0,0 +1,894 @@
+        """Build a kNN+MST network from similarity/distance matrix with UMAP layout.
+
+        Returns a dict with nodes (id, label, x, y, community) and links (source, target, weight, similarity, kind).
+        """
+        import numpy as np
+        import networkx as nx
+        import umap
+        try:
+            import igraph as ig
+            import leidenalg as la
+            have_leiden = True
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)